app/views/value.njk

```diff
@@ -31,4 +32,4 @@
       </div>
     </div>
   </div>
-{% endblock %}
+{% endblock %}
```

Comment:
as above.

Reply:
There is no extra line in the code, though it looks that way on the snippet.